apps/flow.md

```diff
@@ -0,0 +1,46 @@
+# Flow
+How to be Good at Flow
+
+## Terms
+
+### Workspaces
+Workspaces contain all of the info for large projects where clients and contractors are allowed to see our process in action. These are the top level of the Flow hierarchy.
+
+### Projects
+Projects contain all of the info for a subset of a project in a workspace. For Good, this is how we split our internal projects from each other. For larger projects, this will typically be an Active Sprint project and a Backlog project.
```

Comment:
I think this definition covers our larger projects well, but isn't general enough to encompass smaller projects. In some cases, a Project is really just that — an _entire_ project, no just a subset of a larger project.

A better definition would be something like this:

"Depending on the size and scope of what we are working on, a Project can be either an entire project (_i.e. ACME Marketing Site_) or a deliverable subset of a larger project (_i.e. Marketing Site for ACME App_, where ACME App may have its own Workspace."

^you can change that however you'd like, but hopefully that at least gets my point across.